ledger/participant-integration-api/src/main/scala/platform/apiserver/services/logging/package.scala

```diff
@@ -91,6 +91,12 @@ package object logging {
   private[services] def workflowId(id: String): LoggingEntry =
     "workflowId" -> id
 
+  private[services] def packageId(id: String): LoggingEntry =
+    "packageId" -> id
+
+  private[services] def traceId(id: Option[String]): LoggingEntry =
```

Comment:
What's the difference between this one and the other one that you just introduced [here](https://github.com/digital-asset/daml/pull/15995/files#diff-c877e2c32219d2bbf486172296c6af3b44490410f1bdd52db0163a76f911c8f4R10) ?

Reply:
You are right I am removing the duplication for `traceId`.

Reply:
Or should I move all of them there?

Reply:
I would do what Tudor has suggested, i.e. one definition of traceId in common, other definitions remain where they are